### PR TITLE
Contrib/concentric ai

### PR DESCRIPTION
**Pull request by shams-concentricai**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Contains the updated Readme file along with an image of the Playbook.

## Description
To update the Readme file and provide more information in the marketplace.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [X] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by esharf** [comment]
@shams-concentricai did you fill the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)?
if not please do so.

**Comment by shams-concentricai** [comment]
@esharf , I have now filled out the contribution form ! Please let me know if something else needs attention as well.

File: Tests/conf.json [hunk 1]
```diff
@@ -1474,6 +1474,10 @@
             "memory_threshold": 200,
             "instance_names": "use_python_requests_handler"
         },
+        {
+            "integrations": "ConcentricAI",
+            "playbookID": "ConcentricAI Demo Playbook"
+        },
```
Comment (bakatzir) [inline Tests/conf.json:1480]: No need, as we do not run this test in our nightly.
```suggestion
```

**Review by bakatzir — COMMENTED** [review]
please add rn using 

`demisto-sdk update-release-notes -i <path> -u documentation`

File: Tests/conf.json (hunk 1 above)
Comment (shams-concentricai) [reply Tests/conf.json:1487]: @bakatzir could you please elaborate ?

Comment (bakatzir) [reply Tests/conf.json:1487]: Indeed. you can add to the integration yml:
```
tests:
- No tests
```
We are working to fix it in the next demisto-sdk release, as it is not required of partners.

Comment (shams-concentricai) [reply Tests/conf.json]: Done.

**Review by bakatzir — APPROVED** [review]

**Pull request merged**